test_runner/fixtures/utils.py

```diff
@@ -26,34 +27,97 @@ def get_self_dir() -> Path:
     return Path(__file__).resolve().parent
 
 
-def subprocess_capture(capture_dir: Path, cmd: List[str], **kwargs: Any) -> str:
-    """Run a process and capture its output
+def subprocess_capture(
+    capture_dir: Path,
+    cmd: List[str],
+    *,
+    check=False,
+    echo_stderr=False,
+    echo_stdout=False,
+    capture_stdout=False,
+    **kwargs: Any,
+) -> Tuple[str, Optional[str], int]:
+    """Run a process and bifurcate its outout to files and log
```

Comment:
outout is probably a typo. Also, the sentence would benefit from a period after "files" (instead of the "and") in "files and log".

Reply:
This was meant to refer to the logger `log` -- I've made it clearer.